### PR TITLE
LEP-49 Pensioner disregard split across client and partner assets

### PR DESCRIPTION
**Pull request by starswan**
https://dsdmoj.atlassian.net/browse/LEP-49

### CODE REVIEW

File: features/pensioner/pensioner_disregard.feature [hunk 1]
```diff
@@ -35,6 +35,31 @@ Feature:
...
+      | percentage_owned            | 100       |
+      | shared_with_housing_assoc   | false     |
+      | subject_matter_of_dispute   | false     |
+    When I retrieve the final assessment
+    Then I should see the following "capital summary" details:
+      | attribute                     | value   |
+      | total_capital                 | 0.0     |
+      | pensioner_capital_disregard   | 80000.0 |
+      | assessed_capital              | 0.0     |
+      | pensioner_disregard_applied   | 0.0     |
```
Comment (exonian) [inline features/pensioner/pensioner_disregard.feature:62]: I am nervous because this test doesn't touch the bug, does it? Do we not need one where some of the disregard is being applied, across both people?

**Review by exonian — COMMENTED** [review]
The code makes sense to me; the tests don't fill me with confidence but it's possible I'm missing something

File: features/pensioner/pensioner_disregard.feature (hunk 1 above)
Comment (starswan) [reply features/pensioner/pensioner_disregard.feature:62]: It does - the scenario is now eligible whereas before it was ineligible. I'll add some extra workings out to show that

Comment (exonian) [reply features/pensioner/pensioner_disregard.feature:62]: Ah ok so this is pensioner disregard applied to partner's capital? Which didn't happen before.
But I was expecting also a test of pensioner disregard applied to both client's capital and partner's capital. I think that would be valuable. I know with our current implementation, if one works the other works, but that's not true of all implementations.

Comment (exonian) [reply features/pensioner/pensioner_disregard.feature:62]: Also I don't actually understand how if `pensioner_disregard_applied` is 0 this has changed, but maybe that will become clear when you show the extra stuff

**Review by exonian — COMMENTED** [review]
I need to understand how the first test, that the `total_capital` and `pensioner_disregard_applied` were both zero, passed. And now the test tests for different values and also passes... 😕 
Was the test originally broken? eg looking at the client numbers instead of the partner ones or something?

**Review by exonian — APPROVED** [review]
Nice fix, good tests
